src/server/api/routers/chat.ts

```
@@ -26,9 +26,9 @@ export const chatRouter = createTRPCRouter({
         const result = await Promise.all(
             all_chats.map(async (chat) => {
                 const clerk_client = await clerkClient()
-                const primary_user_promise = clerk_client.users.getUser(chat.user_ids[0]!)
+                const primary_user_promise = clerk_client.users.getUser(chat.userIds[0]!)
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)